pages/your_tasks/software_documentation.md

```diff
@@ -32,6 +32,7 @@ It helps align the expectations and goals of all parties, while also ensuring th
 * Understanding the purpose allows you to tailor the content accordingly. For example:

```

Comment:
In the Practical guide to Software Management Plans (https://doi.org/10.5281/zenodo.7038280) we call these "User documentation", "Devloper documentation", "Deployment documentation". They don't need to be called the same, but may be useful to link to the guidelines as a reference?

Reply:
I personally like it but I would pass that decision to the EVERSE team

Reply:
I'll have a look an try to incorporate these in the page.